qa/evil-tests/src/test/java/org/elasticsearch/tribe/TribeUnitTests.java

```diff
@@ -59,8 +60,24 @@ public static void createTribes() {
             .put("node.mode", NODE_MODE)
             .put("path.home", createTempDir()).build();
 
-        tribe1 = NodeBuilder.nodeBuilder().settings(Settings.builder().put(baseSettings).put("cluster.name", "tribe1").put("node.name", "tribe1_node")).node();
-        tribe2 = NodeBuilder.nodeBuilder().settings(Settings.builder().put(baseSettings).put("cluster.name", "tribe2").put("node.name", "tribe2_node")).node();
+        tribe1 = NodeBuilder
+            .nodeBuilder()
+            .settings(
+                Settings.builder()
+                    .put(baseSettings)
+                    .put("cluster.name", "tribe1")
+                    .put("node.name", "tribe1_node")
+                    .put(DiscoveryService.SETTING_DISCOVERY_SEED, random().nextLong())
```

Comment:
Hopefully the answer is no, we do this here only so the test is reproducible? but then if that's the case I wonder, how do we test that the setting is not really needed, cause it shouldn't be? :)